### PR TITLE
Updated Proteomics metadata.yml

### PR DESCRIPTION
**Pull request by Stortebecker**
Hi, 
I updated the metadata for several tutorials.
Cheers, Flo

### CODE REVIEW

**Comment by Stortebecker** [comment]
@blankclemens Warum failed denn travis hier?

**Comment by blankclemens** [comment]
Because Travis checks all the links, not only the ones in the files from your PR. And there's a link in `./_site/topics/training/tutorials/create-new-tutorial-metadata/tutorial.html` that fails. 

Pretty common, pretty annoying. If that is solved one should restart the test here.

**Comment by blankclemens** [comment]
Ah this is an older PR, so the link has been fixed. I restarted the build.

**Comment by willdurand** [comment]
@Stortebecker English please :)

File: topics/proteomics/metadata.yaml [hunk 1]
```diff
@@ -103,7 +108,7 @@ material:
         github_username: "bgruening"
```
Comment (bebatut) [inline topics/proteomics/metadata.yaml]: The contributors should be like this now: 

```
contributors:
      - timothygriffin
      - pratikdjagtap
      - jj-umn
      - blankclemens
```
With a list of GitHub username.
Can you change it?

Comment (Stortebecker) [reply topics/proteomics/metadata.yaml]: Done

**Pull request merged**